packages/contracts/docs/core/04-contracts-overview.md

```diff
@@ -47,11 +42,11 @@ To learn more about the architecture of the system, visit the section explaining
 
 #### Permission Related
 
-| Contract            | Description                                                                                          | Relationship                                         |
-| :------------------ | :--------------------------------------------------------------------------------------------------- | :--------------------------------------------------- |
-| `PermissionManager` | A permission manager (an access control list (ACL)) that `DAO` and other contracts can inherit from. | owned by `DAO`                                       |
-| `PermissionLib`     | A library for data types needed for the processing of permission operations.                         | used by `DAO`, `PluginSetup`, `PluginSetupProcessor` |
-| `IPermissionCondition` | An interface for condition implementations.                                                             | implementations can be used by `PermissionManager`   |
+| Contract               | Description                                                                                          | Relationship                                         |
```

Comment:
If this is no longer work in progress and it's intended for the auditors, have we updated all the new contracts here?

Reply:
I would like to remove this table in the next step.
This PR was about the subtask of fixing the `how-it-works/core` section of the docs.

Reply:
Removed in e96bddac032313aacf92d92795f83f37128eb902